src/renderers/webgl/WebGLPrograms.js

```diff
@@ -307,7 +307,7 @@ function WebGLPrograms( renderer, cubemaps, cubeuvmaps, extensions, capabilities
 			toneMapping: material.toneMapped ? renderer.toneMapping : NoToneMapping,
 			useLegacyLights: renderer.useLegacyLights,
 
-			premultipliedAlpha: material.premultipliedAlpha,
+			premultipliedAlpha: material.premultipliedAlpha || HAS_TRANSMISSION,
```

Comment:
I am not sure this is justifiable...

Reply:
Good catch, I think I forgot to revert this.